__tests__/app.test.js

```diff
@@ -62,6 +62,7 @@ describe("/api", () => {
             category: "euro game",
             created_at: expect.any(String), //highlighting this to sort out
             votes: 1,
+            comment_count: "0",
```

Comment:
I would have added a new test for this ticket rather than amending the one that is there